doc/source/whatsnew/v0.4.x.rst

```diff
@@ -1,6 +1,6 @@
 .. _whatsnew_04x:
 
-v.0.4.1 through v0.4.3 (September 25 - October 9, 2011)
+v.0.4.1 through v0.4.3 (september 25 - october 9, 2011)
```

Comment:
I think the months should still be capitalized as proper nouns? cc @datapythonista

Reply:
Yes, afaik in English they should be capitalized.

@cleconte987 do you mind adding all month names to the list of exceptions please?

Reply:
Ok, doing it now.